pom.xml

```diff
@@ -1803,6 +1803,11 @@
                         <nexusUrl>${jboss.repo.nexusUrl}</nexusUrl>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-shade-plugin</artifactId>
+                    <version>3.4.1</version>
```

Comment:
Nitpick: have a property to set the versions rather than explicitly as in here.

Reply:
Don't see this pattern applied consistently here, will still rework this PR. Back to draft mode.